### PR TITLE
Included information about the destination and files progress in the downloads endpoint

### PR DESCRIPTION
**Pull request by devos50**

### CODE REVIEW

**Comment by lfdversluis** [comment]
Maybe you could add a test that captures the bug you just observed?

**Comment by devos50** [comment]
It has been a while since I made this code but apparently, it was a bug in the GUI and not in the Core. I will reword the commit to better reflect what I've done.

**Comment by devos50** [comment]
@lfdversluis can you review?

**Comment by lfdversluis** [comment]
Looks good to me

File: Tribler/Core/Modules/restapi/downloads_endpoint.py [hunk 1]
```diff
@@ -94,11 +96,12 @@ def render_GET(self, request):
                              "speed_down": download.get_current_speed(DOWNLOAD),
                              "speed_up": download.get_current_speed(UPLOAD),
                              "status": dlstatus_strings[download.get_status()],
-                             "size": download.get_length(), "eta": download.network_calc_eta(),
+                             "size": download.get_def().get_length(), "eta": download.network_calc_eta(),
```
Comment (lfdversluis) [inline Tribler/Core/Modules/restapi/downloads_endpoint.py]: maybe place items on a line new line?

Comment (devos50) [inline Tribler/Core/Modules/restapi/downloads_endpoint.py]: I agree, it's a bit more readable with a newline here indeed, will add it when I process @whirm 's feedback

**Comment by devos50** [comment]
@whirm can you review?

**Comment by whirm** [comment]
Looks fine to me

File: Tribler/Core/Modules/restapi/downloads_endpoint.py (hunk 1 above)
Comment (devos50) [inline Tribler/Core/Modules/restapi/downloads_endpoint.py]: @lfdversluis I just noticed that in subsequent lines, I sometimes have multiple (short) items on the same line so I would rather leave it like this to be consistent.

**Comment by devos50** [comment]
@whirm ready for merge

**Pull request merged**